homework-2/Program5.java

```diff
@@ -0,0 +1,58 @@
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+
+class Program5 {
+    public static void main(String[] args) {
+
+        //          1
+
+        try {
+            FileInputStream fin = new FileInputStream("root.txt");
+        } catch (FileNotFoundException e) {
+            System.out.println("Error: " + e);
+        }
+
+        //           3
+
+        try {
+            Integer a = 10, b = 0;
+            Integer c = a/b;
+        } catch (Exception e) {
+            System.out.println("Error: " + e);;
+        }
+
+        // Combined 2 & 5
+
+        try {
+            String s1 = "638842";
+            Integer n = Integer.parseInt(s1);
+        }
+        catch (NumberFormatException e) {
+            System.out.println("Error: " + e);
```

Comment:
The exception message should point out the cause of the exception, or some other feedback to the user rather than just saying what has happened